src/constants.js

```diff
@@ -56,19 +41,6 @@ export const LIMIT_ADDRESS_GENERATION = true;
  */
 export const HATHOR_BIP44_CODE = 280;
 
-/**
- * Server options for the user to choose which one to connect
- */
-export const DEFAULT_SERVERS = [
```

Comment:
I'm thinking if we should leave those constants here for now and mark them as deprecated.

Even though our wallets don't use them directly, maybe someone is using.

What do you think?

Reply:
I think it's a good idea, will do this